ci/pipeline

```diff
@@ -192,15 +192,15 @@ def buildDockerAndLinuxPackages(): Unit = {
     %('sbt, "docker:publishLocal")
 
     val toolPath = pwd/'tools/'packager
-    %('make, "all", "-j")(toolPath)
+    %('make, "clean", "all", "-j")(toolPath)
   }
 }
 
 def testDockerAndLinuxPackages(): Unit = {
   utils.stage("Testing Docker Image, Debian and RedHat Packages") {
     // create test-bed docker images and run the package tests
     val testPath = pwd/'tests/'package
-    %('make, "clean", "test")(testPath)
+    %('make, "test")(testPath)
```

Comment:
I added the `clean`. This causes each build to always pull all Docker images. We did not do this before.

Reply:
I think the clean is fine to have. We can rely on Docker rebuilding the image if it needs to.

Reply:
(now that `make purge` is the one that removes the docker containers)